### ISSUE TITLE
No error is displayed when policy is created with invalid paramaters

### ISSUE THREAD

**Issue by sega-yarkin** [rabbitmq/rabbitmq-management#110]
RabbitMQ 3.6.0.

It can be reproduced by putting some random text in 'Definition' fields and submitting the form. Nothing will happen but in the Network console you can see the server responds with a `400 Bad Request`:

![default](https://cloud.githubusercontent.com/assets/1433049/12432895/06eb8b04-bf0f-11e5-82ad-c6617836ebdd.PNG)

```
{"error":"bad_request","reason":"Validation failed\n\n[{<<\"random\">>,<<\"random\">>}] are not recognised policy settings\n"}
```

**Comment by essen** [comment]
Looking into this, I found another form where nothing happens, the "Add binding to this queue" in a queue's details page. If you put a non-existing exchange name you get a 404 with no message.

**Comment by michaelklishin** [comment]
Fixed in #111, thanks to @sega-yarkin for reporting and @essen for the quick turn-around.
